.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,34 +8,5 @@ In case you're only changing docs, make sure to prefix the pull-request title wi
 Finally, be sure to sign commits with your real name.  Since by opening
 a PR you already have commits, you can add signatures if needed with
 something like `git commit -s --amend`.
--->
```

Comment:
The closing tag for the comment has to stay

Reply:
I've re-pushed now to maintain the original one, since a spurious empty line that caused the diff to show the closing tag from the dropped chunk